### ISSUE TITLE
Why mobile is important - remove

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-mobile-intro#18]
I don't think any of this belongs here:

> Why mobile is important
Mobile has become a central technology.
World-wide, Mobile phone users reached 4.77 billion in 2017, and pass the 5 billion mark by 2019 according to Statista.com. In 2014, nearly 60 percent of the population worldwide already owned a mobile phone (Source).
With such a large portion of the world’s population using smart phones, accessibility barriers on mobile have great impact on people with disabilities. Impacts include access to education, jobs, financial institutions and entertainment.

Rationale:
* The point of this resource is not to convince people that mobile accessibility is important overall.
* "Mobile has become a central technology." is obvious to the audience.
* The statistics don't add anything here, so are distracting clutter.
* General impact is addressed in other Resources, and needn't be included here.

The Tutorials have very specific "Why is this important?", which is different than above.

**Comment by sharronrush** [comment]
I agree that this is not aligned to previous statements about Why is this important - and must be sharpened or removed.
